src/storage/src/hummock/sstable/forward_sstable_iterator.rs

```diff
@@ -117,7 +117,7 @@ impl SstableIterator {
         // When all data are in block cache, it is highly possible that this iterator will stay on a
         // worker thread for a full time. Therefore, we use tokio's unstable API consume_budget to
         // do cooperative scheduling.
-        tokio::task::consume_budget().await;
+        // tokio::task::consume_budget().await;
```

Comment:
Why comment this out?

Reply:
Comment out by accidence. Has reverted the changed.